torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -672,6 +689,42 @@ NvrtcFunction nvrtcCompile(
 #ifndef NDEBUG
   args.push_back("-lineinfo");
 #endif
+  args.push_back("-G");
```

Comment:
Should we always be using `-G`?

Reply:
this was to repro the register issue with cpp test. So I can have something to verify this actually works. Will remove it later.